unity/embed_api_tests/main.cpp

```diff
@@ -1754,12 +1754,8 @@ TEST(mono_custom_attrs_has_attr_can_check_assembly_attribute)
     MonoClass *klassTestAttribute = GetClassHelper(kTestDLLNameSpace, "TestAttribute");
     MonoClass *klassAnotherTestAttribute = GetClassHelper(kTestDLLNameSpace, "AnotherTestAttribute");
 
-    GET_AND_CHECK(customAttrInfo, mono_custom_attrs_from_assembly(g_assembly));
-
-    CHECK(mono_custom_attrs_has_attr(customAttrInfo, klassTestAttribute));
```

Comment:
Is the idea that we won't need to export/support these APIs if we have `mono_assembly_has_attribute`?

Reply:
Yes the idea is we no longer will use these APIs. We don't want to be passing this native struct around when there are better options. I didn't want to include that in this PR as it was large enough. I'll do that in a future PR.